# Translations template for nova.
# Copyright (C) 2014 ORGANIZATION
# This file is distributed under the same license as the nova project.
#
# Translators:
msgid ""
msgstr ""
"Project-Id-Version: nova\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2014-06-18 06:06+0000\n"
"PO-Revision-Date: 2014-06-14 19:30+0000\n"
"Last-Translator: openstackjenkins <jenkins@openstack.org>\n"
"Language-Team: Portuguese (Brazil) (http://www.transifex.com/projects/p/nova/"
"language/pt_BR/)\n"
"Language: pt_BR\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"

#: nova/hooks.py:84
#, python-format
msgid "Fatal Exception running %(name)s pre-hook: %(obj)s"
msgstr ""

#: nova/hooks.py:89
#, python-format
msgid "Exception running %(name)s pre-hook: %(obj)s"
msgstr ""

#: nova/hooks.py:105
#, python-format
msgid "Fatal Exception running %(name)s post-hook: %(obj)s"
msgstr ""

#: nova/hooks.py:110
#, python-format
msgid "Exception running %(name)s post-hook: %(obj)s"
msgstr ""

#: nova/objects/instance_fault.py:86
msgid "Failed to notify cells of instance fault"
msgstr "Falha ao notificar células de falha da instância"

#: nova/openstack/common/excutils.py:76
#, python-format
msgid "Original exception being dropped: %s"
msgstr "Exceção original sendo descartada: %s"

#: nova/openstack/common/excutils.py:105
#, python-format
msgid "Unexpected exception occurred %d time(s)... retrying."
msgstr "Exceção não esperada ocorreu %d vez(es)... tentando novamente."

#: nova/openstack/common/loopingcall.py:89
msgid "in fixed duration looping call"
msgstr "em uma chamada de laço de duração fixa"

#: nova/openstack/common/loopingcall.py:136
msgid "in dynamic looping call"
msgstr "em chamada de laço dinâmico"

#: nova/openstack/common/periodic_task.py:179
#, python-format
msgid "Error during %(full_task_name)s: %(e)s"
msgstr "Erro durante %(full_task_name)s: %(e)s"

#: nova/openstack/common/policy.py:511
#, python-format
msgid "Failed to understand rule %s"
msgstr "Falha ao interpretar regra %s"

#: nova/openstack/common/policy.py:521
#, python-format
msgid "No handler for matches of kind %s"
msgstr "Nenhum manipulador para correspondências do tipo %s"

#: nova/openstack/common/policy.py:791
#, python-format
msgid "Failed to understand rule %r"
msgstr "Falha ao interpretar regra %r"

#: nova/openstack/common/service.py:188
msgid "Exception during rpc cleanup."
msgstr "Exceção durante limpeza de RPC."

#: nova/openstack/common/service.py:276
msgid "Unhandled exception"
msgstr "Exceção não tratada"

#: nova/openstack/common/db/api.py:72
msgid "DB exceeded retry limit."
msgstr ""

#: nova/openstack/common/db/api.py:76
msgid "DB connection error."
msgstr ""

#: nova/openstack/common/db/sqlalchemy/session.py:460
msgid "DB exception wrapped."
msgstr "Exceção de BD incluída."

#: nova/openstack/common/db/sqlalchemy/test_migrations.py:267
#, python-format
msgid "Failed to migrate to version %s on engine %s"
msgstr ""

#: nova/virt/libvirt/utils.py:395
#, python-format
msgid "ignoring unrecognized volume_clear='%s' value"
msgstr ""
